framework/src/main/java/org/apache/drill/test/framework/ConnectionPool.java

```diff
@@ -19,6 +19,7 @@
 
 import com.google.common.collect.Queues;
 //import org.apache.drill.jdbc.Driver;
+//import com.mapr.drill.jdbc41.Driver;
```

Comment:
The import is used for testing.  The line in question is for the Simba driver.  The line above it is for the Apache driver.